app/views/spina/journal/primer_theme/issues/_issue_cover.html.haml

```diff
@@ -0,0 +1,7 @@
+-# locals: issue cover_img size[2]
```

Comment:
The other option would be inlining this partial

Reply:
I have no real preference either way. The partial is used more than once, though.

After seeing the number of inline svg's used in Spina 2.1's ERB, and upon reading that partials can be quite slow, I'm not against inlining.